notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue

```diff
@@ -132,7 +132,7 @@
             v-html="noteVersionContent">
           </div>
         </div>
-        <div v-else-if="noteChildren && noteChildren[0] && !noteChildren[0].children.length">
+        <div v-else-if="noteChildren && noteChildren[0] && !noteChildren[0].children.length || note.draftPage && !note.content">
```

Comment:
use here the new variable suggested in https://github.com/Meeds-io/notes/pull/559/files#r884424716 for `hasDraft` and add another computed variable for `hasEmptyContent` to replace `!note.content`. In addition, you should add another computed variable for `this.noteChildren && this.noteChildren[0] && this.noteChildren[0].children?.length > 0` with name `hasChildren`.

Reply:
done!